.github/workflows/plugins-ci.yml

```diff
@@ -58,4 +58,3 @@ jobs:
       - uses: fastify/github-action-merge-dependabot@v3
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
-          target: minor
```

Comment:
Just last request before merging: I would set `target: major` to provide more expressiveness